src/flameGraph.cpp

```diff
@@ -445,13 +558,30 @@ void FlameGraph::printFooter(std::ostream& out) {
     out << "</svg>\n";
 }
 
+void FlameGraph::printTreeHeader(std::ostream& out, long total, int type) {
+    out << TREE_HEADER;
+    if(type == BACK_TRACE){
+      out << "<ul>Backtrace view, total [sample/counter]: " << total << "</ul>\n";        
+    } else {
+      out << "<ul>Call tree view, total [sample/counter]: " << total << "</ul>\n";
+    }
+    out << "<ul class=\"tree\">\n";
+    out << "<button type='button' onclick='treeView(0)'>++</button><button type='button' onclick='treeView(1)'>--</button>\n";
+    out << "<input type='text' id='search' value='' size='35' onkeypress=\"if(event.keyCode == 13) document.getElementById('searchBtn').click()\">\n";
+    out << "<button type='button' id='searchBtn' onclick='search()'>search</button>\n";
```

Comment:
This can be probably included to TREE_HEADER constant, not to pollute the code too much. If required, the header can be customized with sprinf, like with SVG_HEADER.

Reply:
Yes, I have moved as suggested.